html-css/week1/index.html

```diff
@@ -0,0 +1,166 @@
+  </head>
+  <body id="top">
+    <nav>
+      <input type="checkbox" id="menu-toggle" class="menu-checkbox" />
+      <label for="menu-toggle" class="hamburger-icon">&#9776;</label>
+      <ul class="navigation-list">
+        <li><a class="active" href="#hero">Home</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#skills">Skills</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#hero">Contact</a></li>
+      </ul>
+    </nav>
+
+    <header id="hero">
+      <div class="hero-text">
+        <h1 class="developer-name">
+          Güzide &nbsp; GÜZELBEY &nbsp; ESENGÜN
+        </h1>
+        <h2>Front-end Developer</h2>
+      </div>
+      <div id="contact" class="hero-contact">
+        <ul class="contact-icons">
+          <li>
+            <a
+              class="github"
+              href="https://github.com/GuzideGuzelbey"
+              target="_blank"
+              ><i class="bi bi-github" style="color: #02c9c9;"></i
+            ></a>
+          </li>
+          <li>
+            <a
+              class="linkedin"
+              href="https://www.linkedin.com/in/guzide-guzelbeyesengun/"
+              target="_blank"
+              ><i class="bi bi-linkedin" style="color: #02c9c9;"></i
+            ></a>
+          </li>
+          <li>
+            <a
+              class="fcc"
+              href="https://www.freecodecamp.org/Guzide"
+              target="_blank"
+              ><img src="images/free-code-camp.png" alt="free-code-camp"
+            /></a>
+          </li>
+          <li>
+            <a class="codepen" href="https://codepen.io/guzit" target="_blank"
+              ><img src="images/codepen.png"
+            /></a>
+          </li>
+          <li>
+            <a
+              class="email"
+              href="mailto:guzide.guzelbey@gmail.com"
+              target="_blank"
+              ><i class="bi-regular bi-envelope" style="color: #02c9c9;"></i
+            ></a>
+          </li>
+        </ul>
+      </div>
+    </header>
+
+    <main>
+      <section class="about" id="about">
+        <div class="about-title">
+          <h2 class="about-title-text">About Me</h2>
+        </div>
+        <div class="about-text">
+          <p>
+            Passionate about web development with a strong foundation in
+            industrial design and extensive experience in ergonomics, user
+            interaction, and usability.<br /><br />Eager to leverage skills in
+            HTML, CSS, JavaScript, and React.js to create engaging and
+            user-friendly web applications.<br /><br />Experienced in applying
+            Scrum in project management and dedicated to continuous learning and
+            professional growth in front-end development.
+          </p>
+        </div>
+      </section>
+      <section class="skills" id="skills">
+        <div class="skills-container">
+          <!-- Left Side Skills (Yellow Lines) -->
```

Comment:
remove all  comments

Reply:
Should I also remove the commented sections that I am planning to continue later?